service/workspace/src/workspaceservice.cpp

```diff
@@ -23,9 +23,14 @@ void WorkspaceServiceHandler::getWorkspaces(std::vector<WorkspaceInfo>& _return)
   {
     std::string filename = it->path().filename().native();
 
+    if (!fs::is_directory(filename))
+      // Ignore plain files in the workspace directory - projects are always
```

Comment:
Maybe we can log some warning that the workspace directory contains some irrelevant files.

Reply:
I don't think this should be done... the "workspaceservice" should not have authority over what the webserver loads from there, etc. We'd have to duplicate ignoring "authentication.json", "certificate.pem" and potential other files here.

Reply:
> Maybe we can log some warning that the workspace directory contains some irrelevant files.

The plugin registrar emits warnings in case the **project** directory does not contain the appropriate `project_info.json` - it was only *workspace service* which despite this, sent them name of the project out on the API.